src/actions/editor.js

```diff
@@ -420,6 +400,49 @@ export function sendRecurringData(formValues, contentLanguages, user, updateExis
     }
 }
 
+export function updateRecurringData(superEvent, subEventsToUpdate, contentLanguages, user, superEventId) {
+    return (dispatch) => {
+        const keysNotToUpdate = ['start_time', 'end_time', 'id', 'super_event_type']
+        // update sub events based on super event data
+        const updatedSubEvents = subEventsToUpdate.map(subEvent => {
+            const updatedSubEvent = Object.keys(subEvent)
+                // filter out keys that we don't want to update
+                .filter(key => !keysNotToUpdate.includes(key))
+                .reduce((acc, key) => {
+                    if (!isNil(superEvent[key])) {
+                        acc[key] = superEvent[key]
+                    }
+                    return acc
+                }, subEvent)
+
+            return {...updatedSubEvent, ...nullifyMultiLanguageValues(updatedSubEvent, contentLanguages)}
```

Comment:
Hm, why is the logic different between sending and updating subevents? I.e. why is nullify not called when sending, but it is called when updating? Are `formValues` already nullified when sending, but not nullified when updating?